From c97d8f686fd7f1bb4a9683ec3c7a93991ffb330f Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Thu, 13 Feb 2020 16:23:35 -0600
Subject: [PATCH 033/120] dt-bindings: fpga: add compatible value to Stratix10
 SoC FPGA manager binding

commit de81ee1bbf99cde3036eb77ed054c92020b768d7 from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit eb2ab83abfc4ac20d7bed0c04c41446f9b1258f8]

Add a compatible property value to Stratix10 SoC FPGA manager binding file

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 .../devicetree/bindings/fpga/intel-stratix10-soc-fpga-mgr.txt  | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/fpga/intel-stratix10-soc-fpga-mgr.txt b/Documentation/devicetree/bindings/fpga/intel-stratix10-soc-fpga-mgr.txt
index 6e03f79287fb..0f874137ca46 100644
--- a/Documentation/devicetree/bindings/fpga/intel-stratix10-soc-fpga-mgr.txt
+++ b/Documentation/devicetree/bindings/fpga/intel-stratix10-soc-fpga-mgr.txt
@@ -4,7 +4,8 @@ Required properties:
 The fpga_mgr node has the following mandatory property, must be located under
 firmware/svc node.
 
-- compatible : should contain "intel,stratix10-soc-fpga-mgr"
+- compatible : should contain "intel,stratix10-soc-fpga-mgr" or
+	       "intel,agilex-soc-fpga-mgr"
 
 Example:
 
-- 
2.31.1

